src/modules/RA/ra-modules.js

```diff
@@ -99,8 +99,8 @@ export const resources = [
   {
     name: RES_BASELAYER,
     moduleName: 'BaseLayer',
-    ...baseLayerViews,
     endpoint: 'baselayer',
+    ...baseLayerViews,
```

Comment:
What is the reason behind this change?

Reply:
I did this for the object to match the the ones in the Array
It does not change the behavior, do you want me to revert this change ?

Reply:
Not  necessarily revert the change, but as it is a stylistic change, would be better in a separate commit in my opinion.